app.js

```diff
@@ -33,6 +33,15 @@ app.use('/', reloadRouter);
 app.use(express.static(path.join(cwd)));
 app.use(express.static(path.join(__dirname, 'public')));
 
+var config = fs.existsSync(path.join(cwd, '/config.json')) ?
+    require(path.join(cwd, 'config.json')) :
+    require(path.join(__dirname, '/config.json'));
+if (config.routes) {
+    for(var i in config.routes) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.